src/platform/CMakeLists.txt

```diff
@@ -18,9 +18,9 @@ project(osconfig-platform)
 set(osconfig_platform_files
     ./Log.c
     ./Main.c
-    ./ManagementModule.cpp
-    ./ModulesManager.cpp
-    ./MpiServer.c)
+    ./Module.c
+    ./Platform.c

```

Comment:
>Platform [](http://example.com/codeflow?start=6&length=8)

Platform.c should be merged with Main.c. Names mean the same. Consider doing this and compacting the code further.

However, if Platform.c contains the Module Manager, rename it in place and keep as separate as ModuleManager.c. 

We need the "ModuleManager" name back.